raft-tech/TANF-app
# Hotfix: Add support for Base 64 encoding in JWT_KEY variable

### This pull request changes...

Adds support for base64 encoded strings used for the `JWT_KEY` or `JWT_CERT_TEST` environment variables by updating the appropriate usage in `tdpservice/users/api/utils.py:generate_client_assertion()` to first attempt to base64 decode the variables first before passing to the `jwt.encode()` function

## FUNCTIONALITY
Addresses issue observed in cloud.gov after reverting https://github.com/raft-tech/TANF-app/pull/538 where the builds would fail because of being unable to decode the JWT_KEY that was previously converted to base64 format.

## TO TEST

Confirm that the cloud.gov build succeeds without having to alter any environment variables.

### This pull request is ready to merge when...

  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)
